tur/shark/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Bhaviktutorials/shark
+TERMUX_PKG_DESCRIPTION="🦈Future Of Phishing With less delay!!🦈"
+TERMUX_PKG_LICENSE="BSD-3-Clause"
+TERMUX_PKG_MAINTAINER="@termux-user-repository"
```

Comment:
I'd like to suggest setting maintainer to yourself rather than a generic one. But that's not a must.